app/javascript/packs/application.js

```diff
@@ -40,7 +40,26 @@ document.addEventListener("turbolinks:load", () => {
   initMapbox();
   initAutocomplete();
 
-  $('.owl-carousel').owlCarousel({
```

Comment:
just checking - does this code go directly into this file? not in its own file and then imported?

Reply:
I folowed the Lewagon example. But we can ask someone about it tonight thought